Formula/butterfly-cli-package.rb

```diff
@@ -0,0 +1,24 @@
+class ButterflyCliPackage < Formula
+  desc "Application code transformation tool"
+  homepage "https://github.com/paypal/butterfly"
+  url "https://search.maven.org/remotecontent?filepath=com/paypal/butterfly/butterfly-cli-package/2.2.0/butterfly-cli-package-2.2.0.zip"
+  sha256 "7ae80466c5df8ef6d2c1a55bac3b584c94ce746eee1a4cd2674b458c2cae2d16"
+
+  depends_on :java => "1.7+"
```

Comment:
I know this is a Ruby file, but is there any way to refer to property `maven.compiler.source` [in the root pom.xml file](https://github.com/paypal/butterfly/blob/master/pom.xml#L50)? If this would be too complex it is ok to leave it this way.

Reply:
I guess you could but then we'd have to write a custom method instead of leveraging the homebrew depends_on method.